### ISSUE TITLE
[bug] `<rh-accordion-header>` is missing its `accents` slot

### ISSUE THREAD

**Issue by bennypowers** [RedHat-UX/red-hat-design-system#1309]
### Describe the bug

[the rh-accordion-header docs](https://ux.redhat.com/elements/accordion/code/#rh-accordion-header) say it has an `accents` slot/part pair. it doesn't. it should

### Which Component?

rh-accordion-header

### Steps to reproduce

1. use accents slot on rh-accordion-header
2. nothing happens

### Expected behaviour

slot works, looks amazing. wins awards, etc.

### Demo

[Codepen](https://codepen.io/zeroedin/pen/eYxpdrj?editors=1100)

### Element HTML

```HTML
<rh-accordion id="tests">
  <rh-accordion-header>
    <h4>Test 1</h4>
    <rh-badge slot="accents" color="green">Pass</rh-badge>
  </rh-accordion-header>
  <rh-accordion-panel>...logs or whatever</rh-accordion-panel>
  <rh-accordion-header>
    <h4>Test 2</h4>
    <rh-badge slot="accents" color="red">Fail</rh-badge>
  </rh-accordion-header>
  <rh-accordion-panel>...logs or whatever</rh-accordion-panel>
</rh-accordion>
```

### Operating System (OS)

Linux

### Additional Info

h/t @aabughosh who originally raised the issue

cc @zeroedin who suggested doing the same for rh-tabs

**Comment by bennypowers** [comment]
not directly related, but consider also adding content attributes, so...

```html
<rh-accordion id="tests">
  <rh-accordion-header heading="Test 1"></rh-accordion-header>
  <rh-accordion-panel>...logs or whatever</rh-accordion-panel>
  <rh-accordion-header heading="Test 2"></rh-accordion-header>
  <rh-accordion-panel>...logs or whatever</rh-accordion-panel>
</rh-accordion>
```

**Comment by markcaron** [comment]
Closed by #1505
